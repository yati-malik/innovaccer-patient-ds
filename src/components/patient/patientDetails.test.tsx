import React from "react";
import { render, fireEvent } from "@testing-library/react";
import PatientDetails from "./patientDetails";

describe("<PatientDetails />", () => {
  it("renders patient details", () => {
    const patient = {
      patient_id: 1,
      name: "YATI",
      age: 20,
      gender: "Male",
      contact: "YATI@GMAIL",
      city: "NOIDA",
      state: "UP",
      country: "INDIA",
    };
    const patientDetails = render(<PatientDetails patient={patient} />);
    expect(patientDetails).toMatchSnapshot();
  });
});
