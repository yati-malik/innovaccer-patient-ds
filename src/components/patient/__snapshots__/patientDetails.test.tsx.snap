// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PatientDetails /> renders patient details 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="Card Card--shadow-medium"
      >
        <div
          class="Row mt-5"
          data-test="DesignSystem-Row"
        >
          <div
            class="Col Col--m-5 Col--l-5 Col--xl-5 ml-5 mb-5"
          >
            <div
              class="Label Label--withInput"
              data-test="DesignSystem-Label"
            >
              <label
                class="Label-label"
              >
                Name
              </label>
            </div>
            <div
              class="Input Input--regular Input--disabled"
              data-test="DesignSystem-InputWrapper"
              style="min-width: 256px;"
            >
              <input
                class="Input-input Input-input--regular"
                data-test="DesignSystem-Input"
                disabled=""
                name="input"
                type="text"
                value="YATI"
              />
            </div>
          </div>
          <div
            class="Col Col--m-5 Col--l-5 Col--xl-5 ml-5 mb-5"
          >
            <div
              class="Label Label--withInput"
              data-test="DesignSystem-Label"
            >
              <label
                class="Label-label"
              >
                Age
              </label>
            </div>
            <div
              class="Input Input--regular Input--disabled"
              data-test="DesignSystem-InputWrapper"
              style="min-width: 256px;"
            >
              <input
                class="Input-input Input-input--regular"
                data-test="DesignSystem-Input"
                disabled=""
                name="input"
                type="text"
                value="20"
              />
            </div>
          </div>
        </div>
        <div
          class="Row"
          data-test="DesignSystem-Row"
        >
          <div
            class="Col Col--m-5 Col--l-5 Col--xl-5 ml-5 mb-5"
          >
            <div
              class="Label Label--withInput"
              data-test="DesignSystem-Label"
            >
              <label
                class="Label-label"
              >
                Gender
              </label>
            </div>
            <div
              class="Input Input--regular Input--disabled"
              data-test="DesignSystem-InputWrapper"
              style="min-width: 256px;"
            >
              <input
                class="Input-input Input-input--regular"
                data-test="DesignSystem-Input"
                disabled=""
                name="input"
                type="text"
                value="Male"
              />
            </div>
          </div>
          <div
            class="Col Col--m-5 Col--l-5 Col--xl-5 ml-5 mb-5"
          >
            <div
              class="Label Label--withInput"
              data-test="DesignSystem-Label"
            >
              <label
                class="Label-label"
              >
                Contact
              </label>
            </div>
            <div
              class="Input Input--regular Input--disabled"
              data-test="DesignSystem-InputWrapper"
              style="min-width: 256px;"
            >
              <input
                class="Input-input Input-input--regular"
                data-test="DesignSystem-Input"
                disabled=""
                name="input"
                type="text"
                value="YATI@GMAIL"
              />
            </div>
          </div>
        </div>
        <div
          class="Row"
          data-test="DesignSystem-Row"
        >
          <div
            class="Col Col--m-5 Col--l-5 Col--xl-5 ml-5 mb-5"
          >
            <div
              class="Label Label--withInput"
              data-test="DesignSystem-Label"
            >
              <label
                class="Label-label"
              >
                City
              </label>
            </div>
            <div
              class="Input Input--regular Input--disabled"
              data-test="DesignSystem-InputWrapper"
              style="min-width: 256px;"
            >
              <input
                class="Input-input Input-input--regular"
                data-test="DesignSystem-Input"
                disabled=""
                name="input"
                type="text"
                value="NOIDA"
              />
            </div>
          </div>
          <div
            class="Col Col--m-5 Col--l-5 Col--xl-5 ml-5 mb-5"
          >
            <div
              class="Label Label--withInput"
              data-test="DesignSystem-Label"
            >
              <label
                class="Label-label"
              >
                State
              </label>
            </div>
            <div
              class="Input Input--regular Input--disabled"
              data-test="DesignSystem-InputWrapper"
              style="min-width: 256px;"
            >
              <input
                class="Input-input Input-input--regular"
                data-test="DesignSystem-Input"
                disabled=""
                name="input"
                type="text"
                value="UP"
              />
            </div>
          </div>
        </div>
        <div
          class="Row"
          data-test="DesignSystem-Row"
        >
          <div
            class="Col Col--m-5 Col--l-5 Col--xl-5 ml-5 mb-5"
          >
            <div
              class="Label Label--withInput"
              data-test="DesignSystem-Label"
            >
              <label
                class="Label-label"
              >
                Country
              </label>
            </div>
            <div
              class="Input Input--regular Input--disabled"
              data-test="DesignSystem-InputWrapper"
              style="min-width: 256px;"
            >
              <input
                class="Input-input Input-input--regular"
                data-test="DesignSystem-Input"
                disabled=""
                name="input"
                type="text"
                value="INDIA"
              />
            </div>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="Card Card--shadow-medium"
    >
      <div
        class="Row mt-5"
        data-test="DesignSystem-Row"
      >
        <div
          class="Col Col--m-5 Col--l-5 Col--xl-5 ml-5 mb-5"
        >
          <div
            class="Label Label--withInput"
            data-test="DesignSystem-Label"
          >
            <label
              class="Label-label"
            >
              Name
            </label>
          </div>
          <div
            class="Input Input--regular Input--disabled"
            data-test="DesignSystem-InputWrapper"
            style="min-width: 256px;"
          >
            <input
              class="Input-input Input-input--regular"
              data-test="DesignSystem-Input"
              disabled=""
              name="input"
              type="text"
              value="YATI"
            />
          </div>
        </div>
        <div
          class="Col Col--m-5 Col--l-5 Col--xl-5 ml-5 mb-5"
        >
          <div
            class="Label Label--withInput"
            data-test="DesignSystem-Label"
          >
            <label
              class="Label-label"
            >
              Age
            </label>
          </div>
          <div
            class="Input Input--regular Input--disabled"
            data-test="DesignSystem-InputWrapper"
            style="min-width: 256px;"
          >
            <input
              class="Input-input Input-input--regular"
              data-test="DesignSystem-Input"
              disabled=""
              name="input"
              type="text"
              value="20"
            />
          </div>
        </div>
      </div>
      <div
        class="Row"
        data-test="DesignSystem-Row"
      >
        <div
          class="Col Col--m-5 Col--l-5 Col--xl-5 ml-5 mb-5"
        >
          <div
            class="Label Label--withInput"
            data-test="DesignSystem-Label"
          >
            <label
              class="Label-label"
            >
              Gender
            </label>
          </div>
          <div
            class="Input Input--regular Input--disabled"
            data-test="DesignSystem-InputWrapper"
            style="min-width: 256px;"
          >
            <input
              class="Input-input Input-input--regular"
              data-test="DesignSystem-Input"
              disabled=""
              name="input"
              type="text"
              value="Male"
            />
          </div>
        </div>
        <div
          class="Col Col--m-5 Col--l-5 Col--xl-5 ml-5 mb-5"
        >
          <div
            class="Label Label--withInput"
            data-test="DesignSystem-Label"
          >
            <label
              class="Label-label"
            >
              Contact
            </label>
          </div>
          <div
            class="Input Input--regular Input--disabled"
            data-test="DesignSystem-InputWrapper"
            style="min-width: 256px;"
          >
            <input
              class="Input-input Input-input--regular"
              data-test="DesignSystem-Input"
              disabled=""
              name="input"
              type="text"
              value="YATI@GMAIL"
            />
          </div>
        </div>
      </div>
      <div
        class="Row"
        data-test="DesignSystem-Row"
      >
        <div
          class="Col Col--m-5 Col--l-5 Col--xl-5 ml-5 mb-5"
        >
          <div
            class="Label Label--withInput"
            data-test="DesignSystem-Label"
          >
            <label
              class="Label-label"
            >
              City
            </label>
          </div>
          <div
            class="Input Input--regular Input--disabled"
            data-test="DesignSystem-InputWrapper"
            style="min-width: 256px;"
          >
            <input
              class="Input-input Input-input--regular"
              data-test="DesignSystem-Input"
              disabled=""
              name="input"
              type="text"
              value="NOIDA"
            />
          </div>
        </div>
        <div
          class="Col Col--m-5 Col--l-5 Col--xl-5 ml-5 mb-5"
        >
          <div
            class="Label Label--withInput"
            data-test="DesignSystem-Label"
          >
            <label
              class="Label-label"
            >
              State
            </label>
          </div>
          <div
            class="Input Input--regular Input--disabled"
            data-test="DesignSystem-InputWrapper"
            style="min-width: 256px;"
          >
            <input
              class="Input-input Input-input--regular"
              data-test="DesignSystem-Input"
              disabled=""
              name="input"
              type="text"
              value="UP"
            />
          </div>
        </div>
      </div>
      <div
        class="Row"
        data-test="DesignSystem-Row"
      >
        <div
          class="Col Col--m-5 Col--l-5 Col--xl-5 ml-5 mb-5"
        >
          <div
            class="Label Label--withInput"
            data-test="DesignSystem-Label"
          >
            <label
              class="Label-label"
            >
              Country
            </label>
          </div>
          <div
            class="Input Input--regular Input--disabled"
            data-test="DesignSystem-InputWrapper"
            style="min-width: 256px;"
          >
            <input
              class="Input-input Input-input--regular"
              data-test="DesignSystem-Input"
              disabled=""
              name="input"
              type="text"
              value="INDIA"
            />
          </div>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
